ftest.sh

```diff
@@ -53,7 +53,7 @@ NFS_SERVER=${NFS_SERVER:-${HOSTNAME%%.*}}
 trap 'echo "encountered an unchecked return code, exiting with error"' ERR
 
 IFS=" " read -r -a nodes <<< "${2//,/ }"
-TEST_NODES=$(IFS=","; echo "${nodes[*]:1:8}")
+TEST_NODES=$(IFS=","; echo "${nodes[*]:1:23}")
```

Comment:
Why restrict this at all?
```suggestion
TEST_NODES=$(IFS=","; echo "${nodes[*]:1")
```